config/runtime/baseline-nfs.jinja2

```diff
@@ -0,0 +1,8 @@
+{% set test_method = 'baseline' %}
+{% set boot_commands = 'nfs' %}
```

Comment:
IIUC that line is the only difference with `baseline.jinja2`. It might be worth setting it as a job param and reuse the existing template.

Also, please note that will work only for ChromeBooks (at least for now) as only the  `depthcharge` boot template handles NFS boot with an arbitrary rootfs.